shell/common/rasterizer.cc

```diff
@@ -131,7 +131,14 @@ void Rasterizer::Draw(fml::RefPtr<Pipeline<flutter::LayerTree>> pipeline) {
   // front of the queue and also change the consume status to more available.
   if (raster_status == RasterStatus::kResubmit) {
     auto front_continuation = pipeline->ProduceIfEmpty();
-    front_continuation.Complete(std::move(resubmitted_layer_tree_));
+    if (front_continuation) {
```

Comment:
Can we separate this into another patch please? This bit of the patch does not line up with the changes stated in the commit message.